### PR TITLE
improved css for member panel

### LINKED ISSUE #1096: Update Organization members page
## Description
This page should be fully working and needs some minor front changes
https://projects.invisionapp.com/d/main#/console/18708905/389334921/preview

## Objectives
Minor corrections from: 

to

- [x] Add the org name on top of page
- [x] Icons should be on the right
- [x] Send invitation button centered 
- [x] The add new member field should be larger
- [x] Send invitation snackbar message should be: "Your invitation was sent"
- [x] Add a tip message when clicking on the field send an invitation: "Please enter a valid email address."
- [x] Put an uppercase on the Role
- [x]  Remove Signer column
- [x] Add bold characters following UI screen

### PR DESCRIPTION
**Pull request by Maxstgt**
close #1096 
sccs css cssssssssssssssssssssssss

### CODE REVIEW

File: libs/organization/src/lib/components/member-invitation/member-invitation.component.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+p {
+  padding-left: 10px;
+}
+.bold {
+  font-weight: bold;
+}
```
Comment (ioaNikas) [inline libs/organization/src/lib/components/member-invitation/member-invitation.component.scss]: You can also use `<b></b>` in the html.

File: libs/organization/src/lib/components/member-pending/member-pending.component.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.bold {
+  font-weight: bold;
+  text-transform: capitalize;
+}
```
Comment (ioaNikas) [inline libs/organization/src/lib/components/member-pending/member-pending.component.scss]: If you use it often, you can also create some variables in styles/style.scss file and reuse them

styles.scss
```
.bold {
   font-weight: bold;
}

.capitalize {
  text-transform: capitalize;
}
```

In component
```
<span class="bold capitalize">...</span>
```

Comment (Maxstgt) [reply libs/organization/src/lib/components/member-pending/member-pending.component.scss]: good idea

Comment (Maxstgt) [reply libs/organization/src/lib/components/member-pending/member-pending.component.scss]: I made an Issue

**Pull request merged**